src/main/java/tds/exam/results/validation/impl/XSDBackedTDSReportValidator.java

```diff
@@ -11,37 +12,38 @@
 import javax.xml.validation.Schema;
 import javax.xml.validation.SchemaFactory;
 import javax.xml.validation.Validator;
-import java.io.File;
+import java.io.IOException;
 
+import tds.exam.results.configuration.ExamResultsTransmitterServiceProperties;
 import tds.exam.results.trt.TDSReport;
 import tds.exam.results.validation.TDSReportValidator;
 
 @Component
 public class XSDBackedTDSReportValidator implements TDSReportValidator {
     private final JAXBContext jaxbContext;
     private final Validator validator;
+    private final ExamResultsTransmitterServiceProperties properties;
 
-    @Autowired
-    public XSDBackedTDSReportValidator(final JAXBContext jaxbContext) {
+    XSDBackedTDSReportValidator(final JAXBContext jaxbContext,
+                                @Value("classpath:xsd/TestResultsTransmissionFormat_Schema.xsd") final Resource xsd,
```

Comment:
It is required. :/

Considering that we're not 100% sure that we'll even be able to validate on our side I think I'll leave it as is for now.

The reason I'm not sure we'll be able to validate it is because we send a partially completed TRT to TIS.  The TIS application scores the overall exam and then sends it to Reporting.  This is existing behavior in the system.  Now, maybe those parts aren't required, but we'll know more once we have a fully populated TRT